css/style.css

```diff
@@ -1,19 +1,123 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
-body {
+/* Body */
+body h1{
     font-family: 'Roboto', sans-serif;
```

Comment:
A good approach is to set a default font for the page (now only h1 tag inside body is modified)
What would be the right way to do it?